ci-operator/config/openshift/local-storage-operator/openshift-local-storage-operator-master.yaml

```diff
@@ -31,15 +31,15 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-openshift-4.6
-  openshift_release_golang-1.13:
+  openshift_release_golang-1.16:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.16
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.16
```

Comment:
This is the golang that's used for unit tests. I am not sure about the other images above, they may be actually unused and everybody is afraid to remove them. IMO if you change just this `build_root:` golang to 1.16, it should be enough to get the 1.16 into unit tests and leave the rest as it is.

(someone should clean them up, but that can be in a following PR)

Reply:
done